redash/query_runner/__init__.py

```diff
@@ -234,6 +234,13 @@ def fetch_columns(self, columns):
     def get_schema(self, get_stats=False):
         raise NotSupported()
 
+    def _handle_run_query_error(self, error):
+        if error is None:
+            return
+
+        logger.error(error)
+        raise Exception(f"Error during query execution. Reason: {error}")
```

Comment:
I wonder if the `Error during query execution` prefix really needed or it might be confusing? (if the error happens when trying to connect, for example)

Reply:
Well, it used to be "Failed getting schema." which covered only specific query to retrieve a schema. However, now this method is being used to handle errors for the run_query method, which could run different types of queries. so I figured for readability it would be nice to specify the action which led to the error. 
For the db connect issue it will still make sense, because technically the error happened during the query operation. However, I'm sure in case of db disconnect we should handle it in a deeper layer and not care during each individual query run.
Let me know what you think.

Reply:
it also aligns with the vision to split `message` and `details` so we can report properly the error to the user. 

I imagine there could be other types of errors like validation, pre\post checks and so on, that is why this prefix would be useful here.

<img width="614" alt="image" src="https://user-images.githubusercontent.com/2346916/210101347-3c4ea71f-d7b0-4e6b-911e-402b2fd3b701.png">